xbmc/utils/LabelFormatter.cpp

```diff
@@ -258,6 +258,9 @@ std::string CLabelFormatter::GetMaskContent(const CMaskString &mask, const CFile
   case 'C': // programs count
     value = StringUtils::Format("%i", item->m_iprogramCount);
     break;
+  case 'c': // relevance
```

Comment:
The useage of 'c' should be mentioned in the methods comments:
https://github.com/Pedrock/xbmc/blob/ActorSorting/xbmc/utils/LabelFormatter.cpp#L103